x-pack/plugin/spatial/src/test/java/org/apache/lucene/geo/XShapeTestUtil.java

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/*
```

Comment:
not sure what to do here for now? This class is like `GeoTestUtil` but for cartesian geometries. It currently resides in lucene's `sandbox` module; which does not provide a test jar. There is an ongoing bikeshed debate regarding where to place this. Until that is resolved it is useful to have in the spatial module but we need to provide proper attribution.

Reply:
Curious what @rjernst might recommend. Gradle `precommit` fails w/  "invalid license header" if anything other than the "Elastic BV" license is used. If we're allowing Apache 2 licenses do we need to update forbidden checks in the build?

Reply:
You can add an exclusion for the file in the build.gradle for x-pack/plugin/spatial:

```
licenseHeaders {
    // This class was sourced from apache lucene's sandbox module tests
    excludes << 'org/apache/lucene/geo/XShapeTestUtil.java'
}
```

Reply:
This worked great. Thanks @rjernst